src/main/java/vimification/taskui/MainScreen.java

```diff
@@ -55,59 +92,46 @@ private void handleKeyPressed(KeyEvent event) {
         boolean isKeyPressedColon = colonKey.match(event);
 
         if (isKeyPressedColon) {
-            handleCommand();
+            loadCommandInputComponent();
             return;
         }
 
         switch (event.getText()) {
         case "i":
-            System.out.println("You've created a task!");
-            handleTaskCreation();
-            break;
-        case "d":
-            System.out.println("You've deleted a task!");
-            break;
-        case "h":
-            System.out.println("You've moved to the left");
-            break;
-        case "l":
-            System.out.println("You've moved to the right");
+            loadTaskCreationPanelComponent();
             break;
         case "j":
-            System.out.println("You've moved up");
-            break;
         case "k":
-            System.out.println("You've moved down");
+            taskListPanel.requestFocus();
             break;
         }
     }
```

Comment:
I may refactor this to use KeyCode - it is more consistent (we consider KeyCode in the entire codebase).